resources/europe/france/fr-mastodon.json

```
@@ -0,0 +1,17 @@
+{
+  "id": "OSMFR-Mastodon",
+  "type": "mastodon",
+  "account": "osm_fr",
+  "locationSet": {"include": ["fr"]},
+  "languageCodes": ["fr"],
+  "order": 10,
+  "strings": {
+    "community": "OpenStreetMap",
```

```suggestion
    "community": "OpenStreetMap France",
```
Change community details (incorrect)